src/ds/d500/d500-private.cpp

```diff
@@ -29,11 +29,13 @@ namespace librealsense
                         break;
                     }*/
 
+                    /*
```

Comment:
Merge the two comments together so future developers will understand the uncomment message above regards this whole section.

Reply:
Merged.